src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectSettingsAjaxController.java

```diff
@@ -32,9 +34,9 @@
 @Controller
 @RequestMapping("/ajax/projects/{projectId}/settings")
 public class ProjectSettingsAjaxController {
-	private ProjectService projectService;
-	private UserService userService;
-	private ProjectOwnerPermission projectOwnerPermission;
+	private final ProjectService projectService;
```

Comment:
Looks like these final variables were added by the editor

Reply:
I added those. Services are ok to be final since they are autowired by Spring and will not change during the life cycle.